metricbeat/docs/fields.asciidoc

```diff
@@ -3020,6 +3020,53 @@ format: bytes
 
 
 
+[float]
+== pending_tasks fields
+
+pending_tasks
+
+
+
+[float]
+=== `elasticsearch.pending_tasks.max_time_in_queue_millis`
```

Comment:
As these are the pending tasks for the cluster I'm wondering if we should put it under `elasticsearch.cluster.pending_tasks`. This does not necessaryl mean we have to rename the metricset. The namespace is configurable for each metricset.